webtau-core/src/main/java/com/twosigma/webtau/reporter/stacktrace/StackTraceUtils.java

```diff
@@ -113,11 +113,9 @@ private static String filterStackTrace(Throwable t, Predicate<String> filter) {
         String stackTrace = renderStackTrace(t);
         List<String> lines = Arrays.asList(stackTrace.split("\n"));
 
-        return lines.get(0) + "\n" +
-                lines.subList(1, lines.size())
-                        .stream()
-                        .filter(filter)
-                        .collect(Collectors.joining("\n"));
+        return lines.stream()
```

Comment:
Why this change?  I assumed we had the lines.get(0) because we didn't want to apply filtration on the message part of the stack trace and now that's no longer the case.  It is of course possible my assumption was incorrect...

Reply:
For one liners it was doing extra '\n'. Then I looked at what filters are being passed. We always check now for the presence of `at` part, which never happens on first line.
This filter function is private and we control use cases. That was justification.

Reply:
Makes sense